### DISCUSSION TITLE
Wrong Moon and Venus possition for ocultation November 09

### DISCUSSION THREAD

**Discussion by gintasd** [Stellarium/stellarium#3497] [General]
Hello,
I wanted to check upcomminig Moon/Venus ocultation and did find that Stellarium shows wrong possitions for the celestial bodys.
I use v23.3, but earlier version shows the same bad possition.
Just for test have tested various different coordinates - relation betweeen Moon and Venus does not change.
With the same coordinates https://stellarium-web.org/ shows correct possitions.
![Ocultation](https://github.com/Stellarium/stellarium/assets/55763694/b2b703b5-011b-47cb-98c7-284c7e5d7125)

**Comment by 10110111** [comment]
Please check is this is still so in the latest [weekly snapshot](https://github.com/Stellarium/stellarium-data/releases/tag/weekly-snapshot). I don't reproduce this in current `master`.

![Screenshot_2023-11-06_10-19-34](https://github.com/Stellarium/stellarium/assets/6376882/2977c138-d23d-4a60-95c1-c6c7b37f2c2c)

**Comment by alex-w** [comment]
Please enable atmosphere and check it again.

**Comment by gintasd** [comment]
Hello,

Thanks for support.
I just managed to get it OK.
It must be something was wrong in the settings left from older versions.
Before a new version install, I always did old version uninstall but kept settings.
Did not notice anything wrong until now.
So now I have done compleete uninstall removing settings as well.
After installing again the same 23.3 - everything is OK.

So excuse me for a disturbance.

Best Regards
Gintautas

**Reply by gzotti** [reply]
Most likely geocentric vs. topocentric position.

**Comment by gintasd** [comment]
I will do a test.

Thanks.

**Comment by gintasd** [comment]
Yes, it was exactly because topocentric was switched off.

My bad.

Thanks

Best Regards

On Mon, Nov 6, 2023 at 11:39 AM Gintautas Drazdauskas ***@***.***>
wrote:

> I will do a test.
>
> Thanks.
>
> On Mon, Nov 6, 2023, 11:28 Georg Zotti ***@***.***> wrote:
>
>> Most likely geocentric vs. topocentric position.
>>
>> —
>> Reply to this email directly, view it on GitHub
>> <https://github.com/Stellarium/stellarium/discussions/3497#discussioncomment-7485306>,
>> or unsubscribe
>> <https://github.com/notifications/unsubscribe-auth/ANJOF3UDBVB44IBZXUYXENTYDCUVTAVCNFSM6AAAAAA6665EGKVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM3TIOBVGMYDM>
>> .
>> You are receiving this because you modified the open/close state.Message
>> ID: ***@***.***
>> com>
>>
>

-- 
Pagarbiai,
Gintautas Drazdauskas
